openapi/python-nextgen/test/__init__.py

```diff
@@ -0,0 +1,37 @@
+import scicat_openapi_client as soc
+
+configuration = soc.Configuration(
+    host="http://localhost:3000"
+)
+api_client = soc.ApiClient(configuration)
+
+users = {
+    'admin': soc.CredentialsDto(username='admin', password='am2jf70TPNZsSan'),
+    'ingestor': soc.CredentialsDto(username='ingestor', password='aman'),
+    'proposalIngestor': soc.CredentialsDto(
+        username='proposalIngestor',
+        password='aman'),
+    'archiveManager': soc.CredentialsDto(
+        username='archiveManager',
+        password='aman'),
+    'user1': soc.CredentialsDto(
+        username='user2',
+        password='a609316768619f154ef58db4d847b75e'),
+    'user2': soc.CredentialsDto(
+        username='user2',
+        password='f522d1d715970073a6413474ca0e0f63'),
+    #
+    'nouser': soc.CredentialsDto(
+        username='nouser',
+        password='f522d1d715970073a6413474ca0e0f63'),
```

Comment:
Can we pull the passwords in from configuration or functionalAccounts.json file?